### ISSUE TITLE
Allowing users to express disinterest in model.recommend

### ISSUE THREAD

**Issue by andrewtavis** [andrewtavis/wikirec#33]
This issue is for discussing and potentially implementing a way for users to express disinterest in a title when calling [wikirec.model.recommend](https://github.com/andrewtavis/wikirec/blob/main/src/wikirec/model.py). The general idea now would be to allow users to pass a title with a negation indicator of some kind (ex `"!title"`), in which case the selections given the similarity matrix for the given item would be reversed. 

It would be great to know if the above would be intuitive UI, and an implementation would be welcome!

**Comment by andrewtavis** [comment]
@victle, I figure we can discuss this in here :) v0.2.0 is out, btw! I'll do the example update tomorrow. 

I'd like to go away from anything that's explicitly removing things with an arbitrary value - in reference to your idea of removing the top 10 recommendations for Twilight if someone negates it. Overall I think that the ratings are doing enough for this at the moment, and you've expressed interest in some other issues that I'd put higher on the priority list right now. Let's both think on this for the next few steps and come back to it, but I think that your idea of using negative values for disinterest is a good one that could be implemented simply :) Just so I understand it, that would be multiplying `sim_matrix[i]` by `-1` before the arithmetic? Let me know if that's not what you meant by opposite vectors though 🙃

**Comment by victle** [comment]
You're right, simply removing 10 recommendations does seem a bit archaic. I do agree that the ratings are doing decently well for expressing disinterest, so we can definitely put this on the backburner 😃 . And yep! I believe multiplying by `-1` should be sufficient. My only concern is it might push the similarity matrix into a region we don't want.

**Comment by andrewtavis** [comment]
Let's keep the `-1` implementation in mind then :) I think that it's intuitive enough that it wouldn't distract, and would allow people to further quickly find the opposite recommendations for things (could be really bad) 😄 What would your suggestion be for the UI on it? `!title` or `~title` are the ones that jump out first.
